frontend/src/packages/notification/notification.ts

```diff
@@ -0,0 +1,6 @@
+import { Notification } from './notification.package.js';
+
+const notification = new Notification();
+
+export { type NotificationPayload } from './libs/types/notification-payload.type.js';
```

Comment:
Should follow common reexport convention
```suggestion
export { type NotificationPayload } from './libs/types/.types.js';
```